metomi/rose/date.py

```diff
@@ -146,7 +146,7 @@ def date_parse(self, time_point_str=None):
             time_point_str = self.ref_point_str
         if time_point_str is None or time_point_str == self.STR_NOW:
             time_point = get_timepoint_for_now()
-            time_point.set_time_zone_to_local()
+            time_point.to_local_time_zone()
```

Comment:
```suggestion
```
This line isn't doing anything for 2 reasons:
- `TimePoint`s are now immutable, so this returns a new `TimePoint` without affecting the variable `time_point`
- `get_timepoint_for_now()` already is in the local time zone by default, unless you specify the arg `utc=True`